docker-compose.yaml

```
@@ -138,3 +138,27 @@ services:
       driver: "json-file"
       options:
         max-size: 50m
+
+  postgres-exporter_orchestrator-db:
+    image: prometheuscommunity/postgres-exporter
+    ports:
+      - '9187:9187'
+    environment:
+      DATA_SOURCE_URI: ${ORCHESTRATOR_DB_HOST}:${ORCHESTRATOR_DB_PORT}/orchestrator-db?sslmode=disable
+      DATA_SOURCE_USER: ${ORCHESTRATOR_DB_USER}
+      DATA_SOURCE_PASS: ${ORCHESTRATOR_DB_PASSWORD}
+    depends_on:
+      - orchestrator-db
+      - prometheus
+
+  postgres-exporter_telegram-bot-db:
+    image: prometheuscommunity/postgres-exporter
+    ports:
+      - '9188:9187'
```

у нас везде сделано так, что порт, который слушает приложение внутри контейнера совпадает с портом, на котором сидит контейнер, принципиально это не важно, но ради единообразия
```suggestion
      - '9188:9188'
```

---

тогда работать не будет, проверял)